go/test/endtoend/vreplication/cluster_test.go

```diff
@@ -323,6 +323,9 @@ func init() {
 func NewVitessCluster(t *testing.T, name string, cellNames []string, clusterConfig *ClusterConfig) *VitessCluster {
 	vc := &VitessCluster{Name: name, Cells: make(map[string]*Cell), ClusterConfig: clusterConfig}
 	require.NotNil(t, vc)
+
+	vc.CleanupDataroot(t)
```

Comment:
Don't we need to do this in teardown? I thought each workflow (each test run) used its own unique VTDATAROOT, so this new root will already be empty, no?

Reply:
Right, I added to teardown. I also wanted to make sure that the vtdataroot was actually empty: while we should always start with a new random directory, was wondering if during the failures we somehow get an existing vtdataroot. Will look for that the next time I see a "disk space not available". But it shouldn't hurt to keep it I guess.